src/main/resources/data.sql

```diff
@@ -648,3 +648,20 @@ insert into address (id, user_id, line1, line2, city, state, zip) values
 ('42f33d30-f3f8-4743-a94e-4db11fdb747d', '008a4215-0b1d-445e-b655-a964039cbb5a', '412 Maple St', null, 'Dowagiac', 'Michigan', '49047'),
 ('579872ec-46f8-46b5-b809-d0724d965f0e', '00963d9b-f884-485e-9455-fcf30c6ac379', '237 Mountain Ter', 'Apt 10', 'Odenville', 'Alabama', '35120'),
 ('95a983d0-ba0e-4f30-afb6-667d4724b253', '00963d9b-f884-485e-9455-fcf30c6ac379', '107 Annettes Ct', null, 'Aydlett', 'North Carolina', '27916');
+
+INSERT INTO payment (id, user_id, card_number, expiry_month, expiry_year, updated) VALUES
+('0001', '008a4215-0b1d-445e-b655-a964039cbb5a', '4532016747591234', 1, 2024, NOW()),
+('0002', '008a4215-0b1d-445e-b655-a964039cbb5a', '6011514433546201', 2, 2025, NOW()),
+('0003', '008a4215-0b1d-445e-b655-a964039cbb5a', '6771549495588452', 3, 2023, NOW()),
```

Comment:
Is this sample data for testing?  Not sure where this is used.

Reply:
This is used by H2, an in memory database, you can access this data by running the app and sending a GET request to the Payments controller.